src/i_video.c

```diff
@@ -790,29 +819,7 @@ void I_FinishUpdate(void)
 
     if (use_limiter)
     {
-        uint64_t target_time = 1000000ull / targetrefresh;
-
-        while (true)
-        {
-            uint64_t current_time = I_GetTimeUS();
-            uint64_t elapsed_time = current_time - frametime_start;
-            uint64_t remaining_time = 0;
-
-            I_CpuPause();
-
-            if (elapsed_time >= target_time)
-            {
-                frametime_start = current_time;
-                break;
-            }
-
-            remaining_time = target_time - elapsed_time;
-
-            if (remaining_time > 1000)
-            {
-                I_Sleep((remaining_time - 1000) / 1000);
-            }
-        }
+        I_WaitUntil(1000000ull / targetrefresh);
```

Comment:
There's an open source tool called [Special K](https://www.special-k.info/) that's advanced quite a bit since the last time I used it. I mention it because it can measure [render latency](https://github.com/fabiangreffrath/woof/assets/56656010/e8e0bd47-86d2-4fc7-8b79-e878bd39d1d8) which I suspect is where you're seeing improvements. I tried to measure any difference between [(1) a previous commit](https://github.com/fabiangreffrath/woof/actions/runs/9303858034) before the apparent latency regression was fixed, [(2) the master branch](https://github.com/fabiangreffrath/woof/actions/runs/9398319907), and (3) this PR. They were all the same, even on an old Sandy Bridge era laptop I pulled out of storage ([data](https://github.com/fabiangreffrath/woof/assets/56656010/51fa5ee8-c4b0-4aa7-aeab-9a0761367f94)). Since you see improvements on your system, would you mind measuring the results for those three cases?

This is the most interesting part of the overlay:
![special_k2](https://github.com/fabiangreffrath/woof/assets/56656010/ff9b2c32-33a2-4a79-acf1-e3b834ad444b)

Reply:
Thanks, I'll check that out.

Reply:
![woof-special-k](https://github.com/fabiangreffrath/woof/assets/108626604/8d4520a8-cc6f-426c-bbd9-38f976924ee7)
@ceski-1 Here are my numbers playing at 2x resolution. I get about 1ms present latency at 3x, and I can match your number playing at 450%. I didn't notice any difference in the latency numbers between the various code revisions. I only tested using the d3d11 backend because it's the only one I could get to work with exclusive fullscreen + Special K.

Reply:
Okay, thanks for testing.

Reply:
Think I've had a bit of a breakthrough on the phantom latency goblins. Fingers crossed